### ISSUE TITLE
Create-React-App Error.

### ISSUE THREAD

**Issue by elaco** [maisano/react-router-transition#64]
I get this error when trying to use `react-router-transition` with a `create-react-app` project.

```
./node_modules/react-router-transition/src/AnimatedRoute.js
Module parse failed: /node_modules/react-router-transition/src/AnimatedRoute.js Unexpected token (16:49)
You may need an appropriate loader to handle this file type.
| }
| 
| const AnimatedRoute = ({ component, path, exact, ...routeTransitionProps }) => (
|   <Route
|     render={({ location, match }) => (
```

i'm not quite sure what the problem is. At first i was thinking it was because of the spread operator but CRA seems to have it enabled. 

Anyone got this before ?

**Comment by jacobangel** [comment]
Rest spread is enabled, but it's likely that node_modules are excluded from your webpack configuration's babelification. 

That's a bug on our part; we should probably point the module file at a transpiled copy which retains the ES6 module declaration.

In the mean time you can work around this by aliasing `react-router-transitions` to `lib/react-router-transition.js`

Config link here: https://webpack.js.org/configuration/resolve/#resolve-alias

**Comment by christill** [comment]
How can I alias it without ejecting the app?

**Comment by elaco** [comment]
while not really answering your question, heres how i did it. You can import it with a relative path
`import { AnimatedSwitch } from '../node_modules/react-router-transition/lib/react-router-transition'`

**Comment by donald-jackson** [comment]
+1 here, not working, and I don't use webpack

**Comment by kvolkovich-sc** [comment]
+1

**Comment by maisano** [comment]
i just published `1.1.0`, which includes @jacobangel's changes. should be fixed now!

**Comment by elaco** [comment]
Yep, that's fixed thanks!
